release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -73,6 +73,10 @@ For more information, see xref:../getting_started/openshift-overview.adoc[Gettin
 [id="ocp-4-11-installation-and-upgrade"]
 === Installation and upgrade
 
+[id="ocp-4-11-installation-and-upgrade-hyperv"]
+==== Support for Microsoft Hyper-V generation version 2
+By default, the installation program now deploys a Microsoft Azure and a Microsoft Azure Stack Hub cluster using Hyper-V generation version 2 virtual machines (VMs). If the installation program detects that the instance type selected for the VMs do not support version 2, it falls back to version 1 for the deployment.
```

Comment:
```suggestion
By default, the installation program now deploys a Microsoft Azure  cluster using Hyper-V generation version 2 virtual machines (VMs). If the installation program detects that the instance type selected for the VMs does not support version 2, it falls back to version 1 for the deployment.
```

Sorry @mjpytlak, I misled you earlier in my Jira comment. We only made this change in Azure--not Azure Stack Hub.